website/addons/wiki/templates/edit.mako

```diff
@@ -74,12 +74,12 @@
                class="${'col-sm-{0}'.format(12 / num_columns)}"
                style="${'' if 'view' in panels_used else 'display: none'}">
               <div class="osf-panel panel panel-default no-border" data-bind="css: { 'no-border reset-height': $root.singleVis() === 'view', 'osf-panel-flex': $root.singleVis() !== 'view' }">
-                <div class="panel-heading wiki-panel-header wiki-single-heading" data-bind="css: { 'osf-panel-heading-flex': $root.singleVis() !== 'view', 'wiki-single-heading': $root.singleVis() === 'view' }">
-                    <div class="row">
-                        <div class="col-sm-4">
-                            <span class="panel-title" > <i class="fa fa-eye"> </i>  View</span>
+                <div class="panel-heading wiki-panel-header wiki-single-heading" data-bind="css: { 'osf-panel-heading-flex': $root.singleVis() !== 'view', 'wiki-single-heading': $root.singleVis() === 'view' }" style=" padding-top: 0; padding-bottom: 0;">
+                    <div class="row wiki-view-icon-container">
+                        <div class="col-lg-4" >
+                            <div class="panel-title" > <i class="fa fa-eye"> </i>  View</div>
```

Comment:
Hmm. It appears to make no UI difference, but because the `panel-title` class applies `inline-block` display, `span` is [technically correct](http://stackoverflow.com/a/1611079) (which, as everyone knows, is [the best kind of correct](https://www.youtube.com/watch?v=hou0lU8WMgo)).